test/dynamo/test_export.py

```diff
@@ -2411,6 +2411,29 @@ def f(x, y):
         buffer = io.BytesIO()
         torch.save(gm, buffer)
 
+        def test_export_with_inline_constrains(self):
+            def f(x):
+                a = x.item()
+                constrain_as_size(a, 4, 7)
+                return torch.empty((a, 4))
+
```

Comment:
Could we also add a check using FileCheck (ex. https://fburl.com/code/4exbqxbl) to make sure the op is in the graph?